### PR TITLE
Make the RSS refer to Kubernetes, not Hugo.

### PR DESCRIPTION
**Pull request by craigbox**

### CODE REVIEW

File: layouts/index.rss.xml [hunk 1]
```diff
@@ -2,16 +2,16 @@
   <channel>
     <title>{{ .Site.Title }} – {{ .Title }}</title>
     <link>{{ .Permalink }}</link>
-    <description>Recent Hugo news from gohugo.io</description>
+    <description>The blog of the Kubernetes project</description>
```
Comment (lucperkins) [inline layouts/index.rss.xml]: I'd change this to "The Kubernetes project blog"

Comment (craigbox) [reply layouts/index.rss.xml]: done

**Pull request merged**